src/draw/DefaultDrawer.js

```diff
@@ -193,66 +210,41 @@ class DefaultDrawer {
   }
 
   /**
-   * Initialize all draw option of the components.
+   * Initializes all draw option of the components.
+   * @param {Object} d3Element - D3 selected elements.
```

Comment:
`D3 selected elements.` -> `D3 selected element.`

Reply:
Done

Reply:
`{Object} d3Elements` shouldn't it be an array ?